src/components/QueryResult.jsx

```diff
@@ -107,7 +108,7 @@ const QueryResult = (props) => {
         return error
     }
     return (
-        <div>
+        <div className="pl-24">
```

Comment:
I re-centered the D3 chart so this shouldn't be needed. See comparison:

- https://serratus.io/query?run=ERR2756788
- https://paginator.serratus.io/query?run=ERR2756788